src/js/polyfill/Array.prototype.find.js

```diff
@@ -0,0 +1,46 @@
+// https://tc39.github.io/ecma262/#sec-array.prototype.find
```

Comment:
A couple of concerns. 

1. How is this code licensed and do we have explicit permission to reuse it?

2. I'm not sure what this is fixing in terms of a user experience issue

Reply:
1. Given the similar comment headers, I believe it's the same origin as as the `findIndex.js` polyfill that is currently in use (MDN).  I can do some checking if needed.
2. The releases page is broken for IE users.

Reply:
Per https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses code samples and snippets added to MDN after 2010 (which the `Array.prototype.find` polyfill was) are in the public domain and require no attribution.